README.md

```diff
@@ -171,6 +171,17 @@ Use the following steps to prepare your workflow for running on your EC2 self-ho
 4. Create a new EC2 image (AMI) from the instance.
 5. Remove the instance if not required anymore after the image is created.
 
+Alternatively, you can use a vanilla EC2 AMI and setup the dependencies setup via `pre-runner-script` in the workflow YAML file:
+```yaml
+          - name: Start EC2 runner
+            with:
+              mode: start
+              ec2-image-id: ami-07bdb09c49774f92e
+              pre-runner-script: |
+                 sudo yum update -y && \
+                 sudo yum install docker git libicu -y && \
+                 sudo systemctl enable docker
+```
```

Comment:
What if we say like the following, to not give the user the same example two times on the page?

```suggestion
Alternatively, you can use a vanilla EC2 AMI and set up the dependencies via `pre-runner-script` in the workflow YAML file. See example in the `pre-runner-script` documentation below.
```